compiler/modules/front.syntax/parse.bal

```diff
@@ -133,38 +137,53 @@ function parseModuleDecl(Tokenizer tok, ModulePart part) returns ModuleLevelDefn
 }
 
 function parseTypeDefinition(Tokenizer tok, ModulePart part, Visibility vis) returns TypeDefn|err:Syntax {
```

Comment:
You should parse in the startPos of the declaration as a parameter.

Reply:
fixed